### ISSUE TITLE
[Quality Of Life] Use 'package_info_plus' to update version number in settings

### ISSUE THREAD

**Issue by Ranger-NF** [Demizo/Daily_You#92]
I noticed that you were changing the version number manually on every update. You could use [package_info_plus](https://pub.dev/packages/package_info_plus) to do that for you. As of right now, there is only a single place where the version number is shown, so its up to you if you want to use it or not :)

**Comment by Demizo** [comment]
Agreed, I plan to automate the version codes in all the files at some point.
